src/radical/saga/adaptors/slurm/slurm_job.py

```diff
@@ -606,11 +609,10 @@ def _job_run(self, jd):
                 # gres resources are specified *per node*
                 assert(n_nodes), 'need unique number of cores per node'
 
-                if cpu_arch: gpu_arch = cpu_arch.lower()
-                else       : gpu_arch = 'p100'
-                if gpu_arch == 'k80': count = 4
-                else                : count = 2
-                script += "#SBATCH --gres=gpu:%s:%d\n" % (gpu_arch, count)  # Make sure we take a full GPU node
+                gpu_arch = 'p100' if not gpu_arch else gpu_arch.lower()
```

Comment:
This is correct, but I don't think it makes the code clearer.

Reply:
it is pretty standard and straightforward construction (reduces space and fulfill pylint/flake-requirements), but ok I'll return previous one back

Reply:
*sigh* I know, thanks for bearing with me.